### DISCUSSION TITLE
Suggestion: No overlap signal with pedestrian and vehicle signals

### DISCUSSION THREAD

**Discussion by waterlavas104** [slyh/Cities2-TrafficLightsEnhancement#38] [Ideas]
Is that possible to have split phrasing signals that both pedestrian and vehicle signals won’t overlap each other, which mean the turning vehicle no need to yield to the crossing of pedestrian as the pedestrian signal won’t be green unless there’s no any incoming direction of vehicle going through.

**Comment by slyh** [comment]
It is planned to rewrite the pedestrian signal to better align with user expectations.
